tests/unit/components/pacbio/PacbioWellModal.spec.js

```diff
@@ -108,9 +108,16 @@ describe('PacbioRunInfo', () => {
       expect(modal.mutateWell).toBeCalledWith({ position: props.position, property: 'sequencing_mode', with: 'CLR' })
     })
 
-    it('updateExtensionTime', () => {
-      modal.updateExtensionTime('1')
-      expect(modal.mutateWell).toBeCalledWith({ position: props.position, property: 'extension_time', with: '1' })
+    it('updatePreExtensionTime', () => {
+      modal.updatePreExtensionTime('1')
+      expect(modal.mutateWell).toBeCalledWith({ position: props.position, property: 'pre_extension_time', with: '1' })
```

Comment:
Like Katy said above, making these all integers for consistency would be a good idea. And either 0/2 depending on the mode